tests/auth_tests/test_forms.py

```diff
@@ -655,6 +665,17 @@ def test_validates_password(self):
             form["new_password2"].errors,
         )
 
+    def test_one_of_fields_was_not_given(self):
+        user = User.objects.get(username="testclient")
+        data = {
+            "new_password1": "new-password",
+        }
+        form = SetPasswordForm(user, data)
+        self.assertFalse(form.is_valid())
+        self.assertEqual(
+            form["new_password2"].errors, [Field.default_error_messages["required"]]
+        )
+
```

Comment:
It may be worth also including a test for the opposite.

Reply:
Of course! I like that!

What do you think about a `test_validate_fields` that would be a combination between  `AdminPasswordChangeFormTesttest_missing_passwords` and `AdminPasswordChangeFormTest.test_one_password`?

Reply:
I added an assertion when data is empty. IMO there is no need for more tests.